src/foam/nanos/menu/Menu.js

```diff
@@ -53,7 +53,6 @@
       documentation: 'View initialized when menu is launched.',
       view: {
         class: 'foam.u2.view.FObjectView',
-        allowCustom: true,
```

Comment:
Don't we need this?

Reply:
Same as above.

Reply:
But it only sets allowCustom: true if there are no choices, but there are choices in this case.